ui/package.json

```diff
@@ -54,6 +55,7 @@
     "query-string": "^5.1.0",
     "react": "^16.6.0",
     "react-ace": "8.0.0",
+    "react-bootstrap": "1.0.0-beta.17",
```

Comment:
I guess its not needed?

Reply:
yes you are right. I am still doing some refactoring to the code, and two packages that the library was importing are not used anymore, therefore in the next push will be removed.